starter_code/elevator.js

```diff
@@ -2,18 +2,31 @@ class Elevator {
   constructor(){
     this.floor      = 0;
     this.MAXFLOOR   = 10;
-    this.requests   = [];
+    this.requests   = [Person];
```

Comment:
You can't do this. It should be an empty array where you will push requests. (Person is not existing here [and is a constructor])